src/js/components/Institutions.jsx

```diff
@@ -109,26 +155,47 @@ export default class Institution extends Component {
         pathname={this.props.location.pathname}
         userName={this.props.user.profile.name} />
       <div id="main-content" className="usa-grid">
-        <UserHeading period="2017" userName={this.props.user.profile.name} />
+        <UserHeading
+          period="2017"
+          userName={this.props.user.profile.name} />
         <div className="usa-width-two-thirds">
           {this.props.filings.map((filingObj, i) => {
             const filing = filingObj.filing
+            const latestSubmissionStatus = filingObj.submissions[0].status || null
```

Comment:
Yeah, I did have the same thought ... it has been consistently that way but good to ask.